cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -1065,26 +1065,26 @@ func (s *Supervisor) onMessage(ctx context.Context, msg *types.MessageData) {
 	}
 
 	if msg.AgentIdentification != nil {
-		newInstanceID, err := ulid.Parse(msg.AgentIdentification.NewInstanceUid)
+		newInstanceID, err := uuid.FromBytes(msg.AgentIdentification.NewInstanceUid)
 		if err != nil {
-			s.logger.Error("Failed to parse instance ULID", zap.Error(err))
-		}
+			s.logger.Error("Failed to parse instance UUID", zap.Error(err))
+		} else {
```

Comment:
Is there a test for this change in behaviour?

Reply:
Good catch, I added a test for this.